homeassistant/components/overkiz/water_heater_entities/domestic_hot_water_production.py

```diff
@@ -0,0 +1,338 @@
+"""Support for DomesticHotWaterProduction."""
+from __future__ import annotations
+
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.water_heater import (
+    STATE_ECO,
+    STATE_HIGH_DEMAND,
+    STATE_PERFORMANCE,
+    WaterHeaterEntity,
+    WaterHeaterEntityFeature,
+)
+from homeassistant.const import ATTR_TEMPERATURE, STATE_OFF, STATE_ON, TEMP_CELSIUS
+
+from ..coordinator import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+OVERKIZ_TO_OPERATION_MODE: dict[str, str] = {
+    OverkizCommandParam.STANDARD: STATE_ON,
+    OverkizCommandParam.HIGH_DEMAND: STATE_HIGH_DEMAND,
+    OverkizCommandParam.STOP: STATE_OFF,
+    OverkizCommandParam.MANUAL_ECO_ACTIVE: STATE_ECO,
+    OverkizCommandParam.MANUAL_ECO_INACTIVE: STATE_OFF,
+    OverkizCommandParam.ECO: STATE_ECO,
+    OverkizCommandParam.AUTO: STATE_ECO,
+    OverkizCommandParam.AUTO_MODE: STATE_ECO,
+    OverkizCommandParam.BOOST: STATE_PERFORMANCE,
+}
+
+OPERATION_MODE_TO_OVERKIZ = {v: k for k, v in OVERKIZ_TO_OPERATION_MODE.items()}
+
+DHWP_AWAY_MODES = [
+    OverkizCommandParam.ABSENCE,
+    OverkizCommandParam.AWAY,
+    OverkizCommandParam.FROSTPROTECTION,
+]
+
+ALTERNATE_WATER_TEMPERATURE: str = "core:WaterTargetTemperatureState"
+ALTERNATE_WATER_SET_TEMPERATURE: str = "setWaterTargetTemperature"
+ALTERNATE_REFRESH_WATER_TEMPERATURE: str = "refreshWaterTargetTemperature"
+ALTERNATE_REFRESH_DHW_MODE: str = "refreshDHWMode"
```

Comment:
Could you use the constants from pyoverkiz for this? Otherwise we need to add these constants.

Reply:
I did add them [a week ago](https://github.com/iMicknl/python-overkiz-api/pull/678) and was waiting a new release to update it.
It's done now ;)